Xamarin.Forms.Core/AutoCapitalization.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Xamarin.Forms
+{
+    public enum AutoCapitalization
```

Comment:
could you add a TypeConverter so it converts CSS values https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform to the right enum ?